Documentation/cmdref/cilium-agent.md

```diff
@@ -18,7 +18,7 @@ cilium-agent [flags]
       --allow-localhost string                               Policy when to allow local stack to reach local endpoints { auto | always | policy } (default "auto")
       --annotate-k8s-node                                    Annotate Kubernetes node (default true)
       --api-rate-limit map                                   API rate limiting configuration (example: --rate-limit endpoint-create=rate-limit:10/m,rate-burst:2) (default map[])
-      --arping-refresh-period duration                       Period for remote node ARP entry refresh (set 0 to disable) (default 5m0s)
+      --arping-refresh-period duration                       Period for remote node ARP entry refresh (set 0 to disable) (default 30s)
```

Comment:
We still set this to 5 minutes in install/kubernetes/cilium/values.yaml. Should that value be updated?

Reply:
Good catch. I would rather remove anything that is explicitly set there so it sticks to agent default. Will fix, thanks!

Reply:
Should be done now, pls double check. Thanks!